src/vos/evtree.c

```diff
@@ -2539,10 +2537,9 @@ evt_node_debug(struct evt_context *tcx, umem_off_t nd_off,
 	if (leaf || cur_level == debug_level || debug_level < 0) {
 		struct evt_rect_df *rect;
 
-		rect = evt_node_mbr_get(nd);
-		D_PRINT("%*snode="DF_X64", lvl=%d, mbr="DF_RECT_DF
+		D_PRINT("%*snode="DF_X64", lvl=%d, mbr="DF_MBR
 			", rect_nr=%d\n", cur_level * EVT_DEBUG_INDENT, "",
-			nd_off, cur_level, DP_RECT_DF(rect), nd->tn_nr);
+			nd_off, cur_level, DP_MBR(nd), nd->tn_nr);
```

Comment:
(style)  'nd' may be misspelled - perhaps 'and'?
(style)  'nd' may be misspelled - perhaps 'and'?
(style)  'nd' may be misspelled - perhaps 'and'?